src/routes/index.js

```diff
@@ -1,9 +1,11 @@
 import express from 'express';
 import userRoutes from './user.route';
 import userwithGoogleRoutes from './user_with_google.route';
+import productRoute from './product.route';
 
 const router = express.Router();
 router.use('/users', userRoutes);
 router.use('/', userwithGoogleRoutes);
 
 export default router;
+export { productRoute, userRoutes };
```

Comment:
Product routes must be also be called here, no need to export them e.g `router.use('/products', productRoutes)`